frontend/device/kobo/powerd.lua

```diff
@@ -112,8 +112,10 @@ function KoboPowerD:init()
     self.autowarmth_job_running = false
 
     if self.device.hasFrontlight() then
-        -- If this device has natural light (currently only KA1)
-        -- use the SysFS interface, and ioctl otherwise.
+        -- If this device has natural light (currently only KA1 & Forma)
+        -- Use the SysFS interface, and ioctl otherwise.
+        -- NOTE: On the Forma, nickel still appear to prefer using ntx_io to handle the FL,
```

Comment:
:rofl:

Reply:
Yeah, I'm starting to believe the blind monkeys may have escaped from the QA department, and managed to infiltrate the dev team, too :D.

Reply:
:see_no_evil: :hear_no_evil: :speak_no_evil:

Reply:
Snark aside, it's impressive how sharp the contrast is between Kobo & lab126 via the glimpse into the development process we get by tinkering with this stuff...

Granted, lab126 can probably afford to throw walls of money at the problem ;).

Reply:
Although at least some of the lack of… finesse is for the better. ;-)

Reply:
Indeed, and let's not forget the generally not being evil part, too ;).

Reply:
Basically, that's just my yearly realization that Amazon still has Amazon problems, and Kobo still has Kobo problems, what was true 8 years ago is mostly still true today ;).

Reply:
I would love to see nickel hardware abstraction layer. 

lab126 is other league. They have the ['guy' who ran linux on a 8-bit micro](http://dmitry.gr/?r=05.Projects&proj=07.%20Linux%20on%208bit)